helm/sematic-grafana-dashboards/templates/sematic_runs_dashboard_configmap.yaml

```diff
@@ -0,0 +1,637 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    grafana_dashboard: "1"
+  name: grafana-dashboard-sematic-runs
+data:
+  sematic_runs_dashboard_configmap.json: |-
+    {
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": {
+              "type": "{{ .Values.datasources.grafana.type }}",
+              "uid": "{{ .Values.datasources.grafana.uid }}"
+            },
+            "enable": true,
+            "hide": true,
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "target": {
+              "limit": 100,
+              "matchAny": false,
+              "tags": [],
+              "type": "dashboard"
+            },
+            "type": "dashboard"
+          }
+        ]
+      },
+      "editable": true,
+      "fiscalYearStartMonth": 0,
+      "graphTooltip": 0,
+      "id": null,
+      "links": [],
+      "liveNow": false,
+      "panels": [
+        {
+          "datasource": {
+            "type": "{{ .Values.datasources.prometheus.type }}",
+            "uid": "{{ .Values.datasources.prometheus.uid }}"
+          },
+          "gridPos": {
+            "h": 3,
+            "w": 24,
+            "x": 0,
+            "y": 0
+          },
+          "id": 3,
+          "options": {
+            "content": "# Run ID\n\n# [$run_id]({{ .Values.sematic_server_dashboard_url }}/runs/$run_id)",
```

Comment:
This means that for any given usage of this dashboard, the link will only be correct if you have *one* Sematic deployment. In our internal usage, changing the namespace also changes what the proper URL is. Obviously if customers are going to use this, it's more important to meet their use case (where there is likely only one Sematic deployment), but it would be nice if we could preserve this feature for ourselves.

Reply:
fixed